index.js

```diff
@@ -124,6 +125,61 @@ app.post('/', function(req, response) {
 				if (error) throw new Error(error);
 				console.log(body);
 			});
+				
+			var options1 =   {
+						method: 'POST',	
+						url: 'https://chatapi.viber.com/pa/send_message',
+						headers: headerBody,
+						body: 
+						{
+							receiver: req.body.sender.id,
+							min_api_version: 1,
+							sender: 
+							{
+								name: 'Susi',
+								avatar: 'https://www.google.co.in/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=0ahUKEwirq6TRoYvTAhWMLI8KHSgVCXsQjRwIBw&url=https%3A%2F%2Fjigyasagrover.wordpress.com%2F2016%2F05%2F26%2Floklak-brief-one-googlesummerofcode-fossasia%2F&bvm=bv.151426398,d.c2I&psig=AFQjCNG01WP05rdho79rkRnGvuwTzs8_hA&ust=1491411117179108' 
+							},
+							tracking_data: 'tracking data',
+							type: 'location',
+							location: {
+							lat: (JSON.parse(body1)).answers[0].actions[2].latitude,
+							lon: (JSON.parse(body1)).answers[0].actions[2].longitude
+							}
+							
+						},
+						json: true 
+					};
+			// request to the chat api of viber.
+			request(options1, function (error, res, body) {
+				if (error) throw new Error(error);
+				console.log(body);
+			});
+			}else{
+			var options =   {
+						method: 'POST',	
+						url: 'https://chatapi.viber.com/pa/send_message',
+						headers: headerBody,
+						body: 
+						{
+							receiver: req.body.sender.id,
+							min_api_version: 1,
+							sender: 
+							{
+								name: 'Susi',
+								avatar: 'https://www.google.co.in/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&cad=rja&uact=8&ved=0ahUKEwirq6TRoYvTAhWMLI8KHSgVCXsQjRwIBw&url=https%3A%2F%2Fjigyasagrover.wordpress.com%2F2016%2F05%2F26%2Floklak-brief-one-googlesummerofcode-fossasia%2F&bvm=bv.151426398,d.c2I&psig=AFQjCNG01WP05rdho79rkRnGvuwTzs8_hA&ust=1491411117179108' 
```

Comment:
Seems like Loklak logo is being used here. Susi has its logo which is being used in Android app as well. I think, it is better to use that.